sympy/polys/tests/test_rings.py

```diff
@@ -1359,10 +1359,46 @@ def test_PolyElement_pdiv():
     q, r = x + y, 0
 
     assert f.pdiv(g) == (q, r)
-    assert f.prem(g) == r
     assert f.pquo(g) == q
     assert f.pexquo(g) == q
```

Comment:
It contains a single test for all the methods.
Should I add more tests for different methods like I did for `prem`

Reply:
Yes